frontend/app/timed-admin/detail/controller.js

```diff
@@ -104,7 +104,8 @@ export default Controller.extend({
       let order = this.store.createRecord('timed-subscription-order', {
         duration: this.duration,
         acknowledged: false,
-        project: this.get('project')
+        project: this.get('project'),
+        ordered: moment(this.get('ordered'))
```

Comment:
If this.ordered can be empty, it might be better to add a check. Otherwise, you'll get the moment for 'now'.